Makefile

```diff
@@ -1,7 +1,18 @@
+.EXPORT_ALL_VARIABLES:
+
 GOLANGCI_LINT_CONFIG = .golangci.yml
 IMAGE_REPOSITORY ?= app-sre
 QUAY_USER ?=
 QUAY_TOKEN ?=
+OPM_VERSION = v1.14.0
+
+BUNDLE_DEPLOY_DIR=deploy/bundle
+MANIFEST_DIR=$(BUNDLE_DEPLOY_DIR)/manifests
+CSV=$(MANIFEST_DIR)/deploymentvalidationoperator.clusterserviceversion.yaml
+BUNDLE_IMAGE?=$(OPERATOR_IMAGE)-bundle
+CATALOG_IMAGE?=$(OPERATOR_IMAGE)-catalog
```

Comment:
I don't understand how this can work if you don't define this two variables after the include... 😅

Reply:
Well, it wouldn't. :). Fixed.